api/v1beta1/disruption_webhook.go

```diff
@@ -460,18 +460,18 @@ func (r *Disruption) initialSafetyNets() ([]string, error) {
 		}
 
 		if r.Spec.Network != nil {
-			if caught := safetyNetNeitherHostNorPort(*r); caught {
-				logger.Debugw("the specified disruption either contains no Hosts or contains a Host which has neither a port nor a host. The more ambiguous, the larger the blast radius.", "SafetyNet Catch", "Network")
+			if caught := safetyNetMissingNetworkFilters(*r); caught {
+				logger.Debugw("the specified disruption either contains no Host or Service filters. This will result in all network traffic being affected.", "SafetyNet Catch", "Network")
```

Comment:
nit: The variable `logger` is shadowing the global `logger` variable of the `v1beta1` package. It will be preferable to rename it to avoid side effects during the execution of unit tests.

Reply:
I don't think we're shadowing the global logger, but directly using it? What side effects will we see?

Reply:
<img width="971" alt="image" src="https://github.com/user-attachments/assets/39ade167-c46f-4e0f-a679-1623bf63cd8e">

Reply:
I made the comment at the wrong plage 🙈 It's line 117

Reply:
oh yes, this is true. But it's taking it directly from the global logger, and it isn't new to this PR, but how we've had that logger used forever. If we want to change that, I'm 100% on board, but it's definitely unrelated to these changes